lib/y-richtext.coffee

```diff
@@ -194,16 +150,14 @@ class YRichText extends BaseClass
             color: "grey"
           @editor.setCursor params
 
-      this.lock_editor_propagation = false
-
   # Apply a delta and return the new position
   # @param delta [Object] a *single* delta (see ot-types for more info)
   # @param position [Integer] start position for the delta, default: 0
   #
   # @return [Integer] the position of the cursor after parsing the delta
-  deltaHelper= (delta, position = 0) =>
+  deltaHelper = (thisObj, delta, position = 0) ->
```

Comment:
These function shouldn't be public; they're inner functions of the
class. Therefore, I removed the ":" for a "=".